.tools/validate.go

```diff
@@ -16,6 +16,12 @@ import (
 var DefaultRules = []ValidateRule{
 	func(c CommitEntry) (vr ValidateResult) {
 		vr.CommitEntry = c
+		if len(strings.Split(c.Parent, " ")) > 1 {
```

Comment:
On Thu, Sep 10, 2015 at 06:26:27AM -0700, Vincent Batts wrote:
> > @@ -16,6 +16,12 @@ import (
> >  var DefaultRules = []ValidateRule{
> >  	func(c CommitEntry) (vr ValidateResult) {
> >  		vr.CommitEntry = c
> > +		if len(strings.Split(c.Parent, " ")) > 1 {
> 
> meh. that would be another separated shell-out…

Why?  Can't you just have:

  … "parents": ["%P"], "abbreviated_parents": ["%p"], …,

in prettyLogFormat, and then:

  c.Parents = strings.Split(c.Parents, " ")
  c.AbbreviatedParents = strings.Split(c.AbbreviatedParents, " ")

after your Unmarshal?